### PR TITLE
Feat - added landing page for e com

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
* Designed the landing page
* Added some custom styles
* See the preview here - https://deploy-preview-2--fast-cart.netlify.app/#

### CODE REVIEW

**Comment by Mohasin-Haque** [comment]
recording


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 98.0.4758.82 on Windows 10
Viewport:  1366 x 649 @1x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-2--fast-cart.netlify.app%2F&os=Windows&os_version=10&browser=Chrome&browser_version=98.0&resolution=1366x649&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-2--fast-cart.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6205fc1b5bc9300008e6a1d9&commentID=6205fcfe74c9644328e776e3&resolution=resolved)

File: styles/nav.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    display: flex;
+}
+
+.navbar-links-ecom li{
+    list-style: none;
+}
+
+.navbar-links-ecom li a{
+    text-decoration: none;
+    color: white;
```
Comment (tushargoyal812) [inline styles/nav.css:27]: use color variables as much as possible

File: styles/nav.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+.navbar-ecom .search-box-ecom{
+    display: flex;
+    margin: auto 0;
+    height: 2rem;
+}
+
+.navbar-ecom .search-box-ecom input{
+    border: none;
+    outline: none;
+    background: #fff;
```
Comment (tushargoyal812) [inline styles/nav.css:45]: here also

File: styles/nav.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    height: 100%;
+    padding: 0 0.6rem;
+    font-size: 1rem;
+    width: 18rem;
+}
+
+.navbar-ecom .search-box-ecom span{
+    color: #333;
+    font-size: 1rem;
+    background: #fff;
```
Comment (tushargoyal812) [inline styles/nav.css:55]: and here

**Review by Rishi05051997 — COMMENTED** [review]
Good To merge

**Review by tushargoyal812 — COMMENTED** [review]
looks good to merge only take care of suggestions

**Pull request merged**